packages/indy-vdr/src/pool/IndyVdrPoolService.ts

```diff
@@ -156,6 +156,13 @@ export class IndyVdrPoolService {
     return Promise.allSettled(this.pools.map((pool) => pool.refreshConnection()))
   }
 
+  /**
+   * Get all pool transactions
+   */
+  public getAllPoolTransactions() {
+    return Promise.allSettled(this.pools.map((pool) => pool.transactions))
```

Comment:
This is not a promise right? So then allSettled is not needed I think?

Reply:
`pool.transactions` returns a promise